Makefile

```diff
@@ -0,0 +1,16 @@
+all: doc example demo
+
+paq:
+	cd src && ./paq.sh > ../nuklear.h
+
+doc: paq
+	$(MAKE) -C doc
+	cd doc && ./build.sh
+
+example: paq
+	$(MAKE) -C example
+
+demo: paq
+	$(MAKE) -C demo/glfw_opengl2
+	$(MAKE) -C demo/glfw_opengl3
```

Comment:
This is not enough. I think on unices all non-Windows demos should be tried to build, but if any fails, it shouldn't stop `make` from proceeding, but instead issue a warning with recommendation for common issues (e.g. `WARN could not build glfw_opengl3 - do you have glfw and libc installed?`).

On Windows this target should try to build all Windows demos with same error handling like I described for unices.